043-deprecate-and-remove-jmxtrans.md

```diff
@@ -0,0 +1,84 @@
+# Deprecate and remove JMX Trans
+
+## Current situation
+
+[JMX Trans](https://github.com/jmxtrans/jmxtrans) is a tool which allows to collect data from JMX endpoints of Java applications and send them to other applications and services.
+Strimzi integrates JMX Trans as part of the `Kafka` custom resource.
+You can configure it in `.spec.jmxTrans` section:
+
+```yaml
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  # ...
+  jmxTrans:
+    outputDefinitions:
+      - outputType: "com.googlecode.jmxtrans.model.output.StdOutWriter"
+        name: "standardOut"
+      - outputType: "com.googlecode.jmxtrans.model.output.GraphiteOutputWriter"
+        host: "mylogstash.com"
+        port: 31028
+        flushDelayInSeconds: 5
+        name: "logstash"
+    kafkaQueries:
+      - targetMBean: "kafka.server:type=BrokerTopicMetrics,name=*"
+        attributes:  ["Count"]
+        outputs: ["standardOut"]
+  # ...
+```
+
+## Issues
+
+The JMX Trans tool seems to be stale.
+The last release is from March 31st 2021 - so more than year and a half ago.
+Because the last release is so old, there are also many CVEs in the different dependencies it uses.
+In addition to that, it is falling behind in other aspects as well.
+For example, while we are moving all our container images to Java 17, JMX Trans does not run on Java 17, so it needs to stick with Java 11.
+
+## Proposal
+
+Since the JMX Trans project does not seem to be developed anymore, we should first deprecate the JMX Trans support and if nothing changes, we should remove the support.
+
+In the first phase - as part of Strimzi 0.33 (currently expected in the second part of December or early January) - we will:
+* Deprecate the `.spec.jmxTrans` API in the Kafka custom resource.
+* Update the docs to indicate that JMX Trans is deprecated.
+* Announce the deprecation to the users as part of the Strimzi 0.33 communication (release notes, change log etc.).
+
+In the second phase - as part of Strimzi 0.35 (currently expected to be release in March or April 2023) - we will:
+* The `.spec.jmxTrans` API in the Kafka custom resource has to remain.
+  But it will stay deprecated and will be ignored by the operator (a warning will be issued if it is present in the custom resource).
+* The Strimzi 0.35 and later will check for existence of the JMX Trans resources and delete them if they would exist.
+  The rest of the operator code related to JMX Trans will be removed.
+* The container image for JMX Trans will be removed.
+* Remove JMX Trans form the docs.
```

Comment:
Maybe I would remove the `we will` and keep just the list and then change the `Remove JMX Trans from the docs` to `The JMX Trans will be removed from the docs`